src/components/Main/Main.tsx

```diff
@@ -76,12 +76,12 @@ async function runSimulation(
 
 const severityDefaults: SeverityTableRow[] = updateSeverityTable(severityData)
 
-const isCountry = (country: string): country is keyof typeof countryAgeDistribution => {
+const isCountry = (country: string): country is keyof typeof countryAgeDistributionData => {
   return countryAgeDistributionData.hasOwnProperty(country)
```

Comment:
`countryAgeDistribution` type doesn't exist. `countryAgeDistributionData` is the correct name.